packages/react-components/src/components/ImageGallery.test.tsx

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { ImageGallery, ImageGalleryImageProps } from './ImageGallery';
+import { render, screen } from '@testing-library/react';
+import { registerIcons } from '@fluentui/react';
+
+describe('ImageGallery default layout tests', () => {
+  beforeAll(() => {
+    registerIcons({
+      icons: {
+        cancel: <></>,
+        download: <></>
+      }
+    });
+  });
+  const titleIconTestId = 'titleIconTestId';
```

Comment:
Does this id make sense to be used for both `imageInfo` and `imageInfo2` would it be better for them to be different so you can compare which one you are comparing against?